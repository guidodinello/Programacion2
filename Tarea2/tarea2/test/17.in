# Implementar longitud
longitud
insertarSegmentoDespues 3 (5,3.4) (8,9.4) (6,2.6)
longitud
Fin
