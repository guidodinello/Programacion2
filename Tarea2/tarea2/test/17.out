1>#  Implementar longitud.
2>Longitud: 0
3>Segmento insertado despuÃ©s de loc.
4>Longitud: 3
5>Fin.
